pom.xml

```diff
@@ -19,129 +19,142 @@
-	<maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
     <dependencies>
-      <dependency>
-  <groupId>org.jenkins-ci.plugins</groupId>
-  <artifactId>git</artifactId>
-  <version>3.9.3</version>
-      </dependency>
-      <dependency>
-  <groupId>org.jenkins-ci.plugins</groupId>
-  <artifactId>google-oauth-plugin</artifactId>
-  <version>0.7</version>
-      </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci</groupId>
-        <artifactId>annotation-indexer</artifactId>
-        <version>1.12</version>
-      </dependency>
-      <dependency>
-	<groupId>org.mockito</groupId>
-	<artifactId>mockito-core</artifactId>
-	<version>2.10.0</version>
-	<scope>test</scope>
-      </dependency>
-      <dependency>
-	<groupId>junit</groupId>
-	<artifactId>junit</artifactId>
-	<version>4.12</version>
-	<scope>test</scope>
-      </dependency>
-      <dependency>
-	<groupId>org.jenkins-ci.plugins</groupId>
-	<artifactId>junit</artifactId>
-	<version>1.3</version>
-      </dependency>
-      <dependency>
-	<groupId>com.google.apis</groupId>
-	<artifactId>google-api-services-container</artifactId>
-	<version>v1-rev57-1.25.0</version>
-      </dependency>
-      <dependency>
-  <groupId>com.google.apis</groupId>
-  <artifactId>google-api-services-compute</artifactId>
-  <version>v1-rev199-1.22.0</version>
-      </dependency>
-      <dependency>
-        <groupId>com.google.apis</groupId>
-        <artifactId>google-api-services-cloudresourcemanager</artifactId>
-        <version>v1-rev535-1.25.0</version>
-      </dependency>
-      <dependency>
-  <groupId>com.google.api-client</groupId>
-  <artifactId>google-api-client</artifactId>
-  <version>1.25.0</version>
-      </dependency>
-      <dependency>
-  <groupId>com.google.http-client</groupId>
-  <artifactId>google-http-client</artifactId>
-  <version>1.25.0</version>
-      </dependency>
-      <dependency>
-  <groupId>com.google.guava</groupId>
-  <artifactId>guava</artifactId>
-  <version>19.0</version>
-      </dependency>
-      <dependency>
-	<groupId>org.yaml</groupId>
-	<artifactId>snakeyaml</artifactId>
-	<version>1.19</version>
-      </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>git</artifactId>
+            <version>3.9.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>google-oauth-plugin</artifactId>
+            <version>0.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>annotation-indexer</artifactId>
+            <version>1.12</version>
+        </dependency>
+        <dependency>
+            <groupId>io.kubernetes</groupId>
+            <artifactId>client-java</artifactId>
```

Comment:
This is the culprit for the build failures I believe because it depends on sundr.io which is a gradle package and requires a certain arguments that are not configured properly when running on windows, specifically the java system path.

See this: https://mvnrepository.com/artifact/io.kubernetes/client-java-api/4.0.0

I was able to compile and test on linux without this locally so why not try removing it and using the jenkins pull request builder to test that rather than spinning up a windows vm which I was about to do.

I don't see any other references to `io.kubernetes` or `client-java` in the code, so is this doing something useful during compilation that I wouldn't find while testing or running locally?

Reply:
Many thanks to you for figuring this out!